cluster/gce/gci/configure-helper.sh

```diff
@@ -2694,9 +2727,9 @@ function main() {
   fi
 
   # generate the controller manager, scheduler and cluster autoscaler tokens here since they are only used on the master.
-  KUBE_CONTROLLER_MANAGER_TOKEN=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
-  KUBE_SCHEDULER_TOKEN=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
-  KUBE_CLUSTER_AUTOSCALER_TOKEN=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+  KUBE_CONTROLLER_MANAGER_TOKEN="$(secure_random 32)"
+  KUBE_SCHEDULER_TOKEN="$(secure_random 32)"
+  KUBE_CLUSTER_AUTOSCALER_TOKEN="$(secure_random 32)"
```

Comment:
The output of the new function is longer & has a different format. Is that a problem?

```
$ for i in $(seq 1 10); do secure_random 32; done
2rh09yQmL2OkCrLBl+NzjFZXfW8p/I1waEcdcInPt74=
V2P0Sx8DbQ2fSHmBlD5TIQ13WnE+krXzSytQdYSavIs=
bash: warning: command substitution: ignored null byte in input
onSYBtYtkZSDiMswwlsZJJyOg07hMnGzq+QBRTlU
bash: warning: command substitution: ignored null byte in input
ujDlqE9CwYkwTVYGhHbYbc9qK4jJg1H2dloz0ghSPA==
lIsXSn4xCUDZm6FMwdsywU25anFsb4da7l6jgfVHl0w=
ZB0ant3bWCDrIo4nomBWSL2T91MNxnSMtIU5U/kRQw==
mcWNml7f0s8/lnI30/6+YscTwPkMcZKQzKwgPF/hh5k=
V5kpzout27rBdxp2tw8MfGcrnmG+J08MaZYBavt40Yg=
ZIbO4M2pQhYW9DMY3kxD4Y8y3rRdGfIVFR50IRwyLmM=
bash: warning: command substitution: ignored null byte in input
n9JL2XTEBra7bNUM6P8d5jbAvmAPa3cIMud9jyxb/Q==
```
vs
```
for i in $(seq 1 10); do dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null; echo; done
g4SCrOKRgSZoNbdCK5kRyTXQhyG5FzdP
LPl5C4OGwjaxQgAsU7UKv57nLEGwURsk
qpZ9s0XxJlpbrVv6fpEpiKyKHYZVF4us
k88N0Ap7ElIsDpEzyqPyHozYUJGYwH41
b0FEzZgpO0SYsik6zSwWk4wYKTYLcYng
reLHdWw4griQ8ilXDms7UZkcLhyV0nGD
o5jgSUGTNumsLB0lxaMLs36004wrBX7o
tW5xFRmIzCtZBq5BA1TOhEEVZGhd9OdG
IGSlrmfazSXXAcaElkJqhlz5UlAkgtkr
YX3Hw8QXyCqsraV3hWH0ZyJy9VIu8y7k
```

Reply:
The output changed from 32 characters to 44 characters. 44 characters is the minimum number of bytes required to carry 32 bytes of entropy base64 encoded. I don't think this should be an issue.

Reply:
The other difference is the `tr -d "=+/"` - does it matter?